examples/models/image_classification/PyPandaResNet.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import division
+from __future__ import print_function
+import os
+import argparse
+import sys
+import base64
+import abc
+import tempfile
+import json
+import time
+import shutil
+import importlib
+from collections import OrderedDict
+from typing import Union, Dict, Optional, Any, List
+
+# Rafiki Dependency
+from rafiki.model.knob import BaseKnob
+from rafiki.constants import ModelDependency
+from rafiki.model.dev import test_model_class
+from rafiki.model import PandaModel, FloatKnob, CategoricalKnob, FixedKnob, IntegerKnob, PolicyKnob, utils
+
+# PyTorch Dependency
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.optim import lr_scheduler
+from torch.autograd import Variable
+import torch.backends.cudnn as cudnn
+import torchvision.transforms as transforms
+from torchvision.models.resnet import resnet101
+from torch.utils.data import Dataset, DataLoader
+
+# Misc Third-party Machine-Learning Dependency
+import numpy as np
+
+# Panda Modules Dependency
+from rafiki.panda.models.PandaTorchBasicModel import PandaTorchBasicModel
+from rafiki.panda.modules.mod_modelslicing.data_loader import data_loader
+from rafiki.panda.modules.mod_modelslicing.utils.utilities import logger, AverageMeter, accuracy, timeSince
+from rafiki.panda.modules.mod_modelslicing.utils.lr_scheduler import GradualWarmupScheduler
+from rafiki.panda.modules.mod_modelslicing.models import upgrade_dynamic_layers, create_sr_scheduler, resnet_imagenet, resnet_cifar
+
+# Misc Third-party Machine-Learning Dependency
+import sklearn.metrics
+
+
+KnobConfig = Dict[str, BaseKnob]
+Knobs = Dict[str, Any]
+Params = Dict[str, Union[str, int, float, np.ndarray]]
+
+
+
+
+class PyPandaResNet(PandaTorchBasicModel):

```

Comment:
If I am not wrong, PandaTorchBasicModel was renamed to TorchModel

Reply:
you can see the renamed class https://github.com/nusdbsystem/singa-easy/blob/master/singa_easy/models/TorchModel.py#L74